typescript/sdk/src/middleware/account/InterchainAccountDeployer.ts

```diff
@@ -47,7 +45,13 @@ export class InterchainAccountDeployer extends HyperlaneRouterDeployer<
     const interchainAccountRouter = await this.deployContract(
       chain,
       'interchainAccountRouter',
-      [config.mailbox, ethers.constants.AddressZero, owner, 50_000],
+      [
```

Comment:
I'm a bit worried about our rollout of these ICAs. Imo we are changing the bytecode and deployment tooling to the new ICAs but we would need to plan the migration to them, otherwise we are "bricking" accessing the current ICAs, no?